catalog/resource_sql_table.go

```diff
@@ -0,0 +1,383 @@
+package catalog
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"reflect"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	clustersApi "github.com/databricks/databricks-sdk-go/service/clusters"
+	"github.com/databricks/terraform-provider-databricks/clusters"
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type SqlColumnInfo struct {
+	Name     string `json:"name"`
+	TypeText string `json:"type_text"`
```

Comment:
could you remind me why we cannot name this simply as `type`?

Reply:
If we ever support the additional ways of specifying type (type_name, json, etc..) which are in the API, having `type` also might be confusing. If you are ok with this i will change to `type`.

Reply:
changed to type